interface/patient_file/summary/add_edit_amendments.php

```diff
@@ -161,113 +146,109 @@ function formValidation() {
+            </div>
+            <div class="col-12">
+                <form action="add_edit_amendments.php" name="add_edit_amendments" id="add_edit_amendments" method="post" onsubmit='return top.restoreSession()'>
+                    <input type="hidden" name="csrf_token_form" value="<?php echo attr(CsrfUtils::collectCsrfToken()); ?>" />
+
+                    <div class="form-group mt-3">
+                        <label><?php echo xlt('Requested Date'); ?></label>
+                        <?php if (! $onlyRead) { ?>
+                            <input type='text' size='10' class='form-control datepicker' name="amendment_date" id="amendment_date"
+                                value='<?php echo $amendment_date ? attr(oeFormatShortDate($amendment_date)) : attr(oeFormatShortDate()); ?>'
+                            />
+                        <?php } else { ?>
+                            <input type='text' size='10' class='form-control' name="amendment_date" id="amendment_date" readonly
+                                value='<?php echo $amendment_date ? attr(oeFormatShortDate($amendment_date)) : attr(oeFormatShortDate()); ?>'
+                            />
+                        <?php } ?>
+                    </div>
+
+                    <div class="form-group mt-3">
+                        <label><?php echo xlt('Requested By'); ?></label>
+                        <?php echo generate_select_list("form_amendment_by", "amendment_from", $amendment_by, 'Amendment Request By', ' ', '', '', '', $customAttributes); ?>
+                    </div>
+
+                    <div class="form-group mt-3">
+                        <label><?php echo xlt('Request Description'); ?></label>
+                        <textarea <?php echo ( $onlyRead ) ? "readonly" : "";  ?> id="desc" class="form-control" name="desc" rows="4" cols="30"><?php
+                        if ($amendment_id) {
+                            echo text($amendment_desc);
+                        } else {
+                            echo "";
+                        } ?>
+                        </textarea>
+                    </div>
+
+                    <div class="form-group mt-3">
+                        <label><?php echo xlt('Request Status'); ?></label>
+                        <?php echo generate_select_list("form_amendment_status", "amendment_status", $amendment_status, 'Amendment Status', ' ', '', '', '', $customAttributes); ?>
+                    </div>
+
+                    <div class="form-group mt-3">
+                        <label><?php echo xlt('Comments'); ?></label>
+                        <textarea <?php echo ( $onlyRead ) ? "readonly" : "";  ?> id="note" class="form-control" name="note" rows="4" cols="30"><?php
+                        if ($amendment_id) {
+                            echo "";
+                        } else {
+                            echo xlt('New amendment request');
+                        } ?>
+                        </textarea>
```

Comment:
This should be using the placeholder attribute instead of having a value set

Reply:
@tywrenn 

I try to remove it and use the placeholder to show the text, but It seems like the textarea value isn't empty. I am not sure what happens and which function to execute assign value.

![image](https://user-images.githubusercontent.com/22230889/87643592-9d2c7f80-c77d-11ea-9767-f8dd3a39fbca.png)